### DISCUSSION TITLE
Welcome to MinimizableView Discussions!

### DISCUSSION THREAD

**Discussion by DominikButz** [DominikButz/MinimizableView#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jinswac** [comment]
When I switched the network wifi to phone then the view disappeared. Is there any way to keep the view always.

**Reply by DominikButz** [reply]
Hi, I can't reproduce this problem. I have installed MinimizableViewExample on a real device (iPhone) and if I swipe down to open command centre switch off wifi, then close the command centre view, the presented content view is still there. however, it is true that the down swipe can cause the mini view to minimise, but it does not disappear. 
If this is an issue, you can try the following. In the example project, you can see that there is a function called "dragUpdated" in the RootView. Modify the first "if" expression by including "&& value.location.y > 200" or whatever value you want. This way, swiping down from the top right corner to bring up the command centre will not minimise the content view.
